fmriprep/cli/run.py

```diff
@@ -26,7 +26,8 @@ def main():
     # CRITICAL Save the config to a file. This is necessary because the execution graph
     # is built as a separate process to keep the memory footprint low. The most
     # straightforward way to communicate with the child process is via the filesystem.
-    config_file = config.execution.work_dir / f"config-{config.execution.run_uuid}.toml"
+    config_file = config.execution.work_dir / config.execution.run_uuid / "config.toml"
+    config_file.parent.mkdir(exist_ok=True, parents=True)
```

Comment:
This is a bit unrelated. Are you 100% there are no side-effects?

Reply:
this change cascaded from the addition of the <run-uuid> subdirectory within the work dir. I think it cleans up the structure a bit, organizing the `config-<run-uuid>.toml` top-level files.